### PR TITLE
Fixed HW-54450: Update Keys and Fields query for processingTime

### PR DESCRIPTION
**Pull request by oleksii-lubianyi-epam**

### CODE REVIEW

File: Tests/GraphQL/HyperwalletTransferMethodConfigurationTests.swift [hunk 1]
```diff
@@ -64,15 +64,37 @@ class HyperwalletTransferMethodConfigurationTests: XCTestCase {
 
         bankAccountProcessingTime = graphQlResponse?.transferMethodTypes(countryCode: "CA", currencyCode: "CAD")?
             .first(where: { $0.code == "BANK_ACCOUNT" })
-            .flatMap { $0.processingTime }
-        XCTAssertNotNil(bankAccountProcessingTime)
-        XCTAssertEqual(bankAccountProcessingTime, "1 - 3 business days")
+            .flatMap { $0.processingTimes?.nodes?.first }
+        XCTAssertNotNil(bankAccountProcessingTime, "The procebankAccountProcessingTimessingTime should not be nil")
```
Comment (jsaini-hw) [inline Tests/GraphQL/HyperwalletTransferMethodConfigurationTests.swift]: Typo in procebankAccountProcessingTimessingTime

File: Tests/GraphQL/HyperwalletTransferMethodConfigurationTests.swift [hunk 2]
```diff
@@ -64,15 +64,37 @@ class HyperwalletTransferMethodConfigurationTests: XCTestCase {
...
+                       "BANK_ACCOUNT",
+                       "The transferMethodType should be BANK_ACCOUNT")
 
         payPalAccountProcessingTime = graphQlResponse?.transferMethodTypes(countryCode: "CA", currencyCode: "CAD")?
             .first(where: { $0.code == "PAYPAL_ACCOUNT" })
-            .flatMap { $0.processingTime }
-        XCTAssertNotNil(payPalAccountProcessingTime)
-        XCTAssertEqual(payPalAccountProcessingTime, "IMMEDIATE")
+            .flatMap { $0.processingTimes?.nodes?.first }
+        XCTAssertNotNil(payPalAccountProcessingTime, "The procebankAccountProcessingTimessingTime should not be nil")
```
Comment (jsaini-hw) [inline Tests/GraphQL/HyperwalletTransferMethodConfigurationTests.swift]: Typo in procebankAccountProcessingTimessingTime

File: Tests/GraphQL/HyperwalletTransferMethodConfigurationTests.swift (hunk 1 above)
Comment (oleksii-lubianyi-epam) [reply Tests/GraphQL/HyperwalletTransferMethodConfigurationTests.swift]: Fixed

File: Tests/GraphQL/HyperwalletTransferMethodConfigurationTests.swift (hunk 2 above)
Comment (oleksii-lubianyi-epam) [reply Tests/GraphQL/HyperwalletTransferMethodConfigurationTests.swift]: Fixed

**Pull request merged**